data/routers/project-router.js

```diff
@@ -0,0 +1,92 @@
+const express = require('express');
+
+const projectDb = require('../helpers/projectModel');
+
+const router = express.Router();
+
+// GET request for all projects
+router.get('/', (req, res) => {
+  projectDb
+    .get()
+    .then(projects => {
+      res.status(200).json(projects);
+    })
+    .catch(err => {
+      console.log(err);
+      res.status(500).json({ error: 'The projects could not be retrieved.' });
+    });
+});
+
+// GET request for project by id
+router.get('/:id', (req, res) => {
+  projectDb.get(req.params.id).then(project => {
+    if (project) {
+      res.status(200).json(project);
+    } else {
+      res.status(404).json({ message: 'Project not found' });
+    }
+  });
+});
+
+// GET request for project actions
+router.get('/:id/actions', (req, res) => {
+  projectDb.getProjectActions(req.params.id)
+    .then(actions => {
+      res.status(200).json(actions)
+    })
+    .catch(err => {
+      console.log(err);
+      res.status(500).json({ error: 'Could not retrieve actions' })
+    });
+});
+
+// POST request to add a project
+router.post('/', (req, res) => {
+  projectDb
+    .insert(req.body)
```

Comment:
Don't forget to add a check to make sure the `req.body` has the required properties '' '
```
if (!req.body.name || !req.body.description) {
    return res.status(400).json({err:"No name or description given o.o"})
}
```